src/pytorch_lightning/utilities/meta.py

```diff
@@ -41,40 +41,6 @@
     # TODO: Removed once merged and released on PyTorch side           #
```

Comment:
```suggestion
    # TODO: Remove once merged and released on PyTorch side            #
```

Reply:
I modified in https://github.com/Lightning-AI/lightning/pull/13763/commits/13b633e7ba5b9d3efa6024d2f2b86dab1dc80130 !